### PR TITLE
Move to Developer Preview

### PR DESCRIPTION
**Pull request by nplasterer**
Update docs to indicate the app is now in Developer Preview.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
-![Test](https://github.com/xmtp/xmtp-android/actions/workflows/test.yml/badge.svg) ![Lint](https://github.com/xmtp/xmtp-android/actions/workflows/lint.yml/badge.svg) ![Status](https://img.shields.io/badge/Project_Status-Pre--Preview-red)
+![Test](https://github.com/xmtp/xmtp-android/actions/workflows/test.yml/badge.svg) ![Lint](https://github.com/xmtp/xmtp-android/actions/workflows/lint.yml/badge.svg) ![Status](https://img.shields.io/badge/Project_Status-Developer_Preview-yellow)
 
 `xmtp-android` provides a Kotlin implementation of an XMTP message API client for use with Android apps.
 
 Use `xmtp-android` to build with XMTP to send messages between blockchain accounts, including DMs, notifications, announcements, and more.
 
-This SDK is in **Pre-Preview** status and ready for you to experiment with.
+This SDK is in **Developer Preview** status and ready for you to experiment with.
```
Comment (jhaaaa) [inline README.md:9]: An edit based on the definition of Dev Preview - is this OK? =)

```suggestion
This SDK is in **Developer Preview** status and ready for you to start building with.
```

File: README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
-However, we do **not** recommend using Pre-Preview software in production apps. Software in this status is likely to change based on feedback.
+However, we do **not** recommend using Developer Preview software in production apps. Software in this status is likely to change based on feedback.
```
Comment (jhaaaa) [inline README.md:11]: An edit based on the definition of Dev Preview - is this OK?

```suggestion
However, we do **not** recommend using Developer Preview software in production apps. Software in this status may change based on feedback.
```

**Review by jhaaaa — COMMENTED** [review]
Added a just a few suggested edits and a question =) - thank you, @nplasterer!

File: README.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
-Specifically, this SDK is currently building out an example app.
+Specifically, this SDK is working to improve performance and test push notifications.
```
[the remark this answers is not in the document]
Comment (nplasterer) [reply README.md:13]: Push notifications should work with the current sdk but we are working to build out a push example and some performance examples like caching, db, etc... So not sure how to word that 😄

**Review by elisealix22 — APPROVED** [review]
Incredible milestone @nplasterer ! 👏 🥳

File: README.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 Use `xmtp-android` to build with XMTP to send messages between blockchain accounts, including DMs, notifications, announcements, and more.
 
-This SDK is in **Pre-Preview** status and ready for you to experiment with.
+This SDK is in **Developer Preview** status and ready for you to start building with.
 
-However, we do **not** recommend using Pre-Preview software in production apps. Software in this status is likely to change based on feedback.
+However, we do **not** recommend using Developer Preview software in production apps. Software in this status may change based on feedback.
 
-Specifically, this SDK is currently building out an example app.
+Specifically, this SDK is working to improve performance and test push notifications.
```
Comment (jhaaaa) [inline README.md:13]: Thank you for this @nplasterer  - how about something like this? =)

```suggestion
Specifically, we are working on providing push notifications and performance optimizations in the example app.
```

Comment (nplasterer) [reply README.md:13]:
```suggestion
Specifically, we are working on providing an example of push notification and performance optimizations in the example app.
```

**Review by jhaaaa — APPROVED** [review]
Thank you for your help with updating the README, @nplasterer - LGTM!

**Pull request merged**